<launch>
    <node name="feature_saver_node" pkg="slytherin_dagger" type="feature_saver.py" output="screen" >
    <param name="bag_folder" type="string" value="/home/icoderaven/bagfiles/" />
    <param name="list_of_files" type="string" value="/home/icoderaven/change.txt" />
    
	<param name="image_topic" type="string" value="/camera/image_raw" />
	<param name="joy_topic" type="string" value="/cmd_vel" />
	<param name="record_topic" type="string" value="/record" />
	
</node>
</launch>
